<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingIERatio</class>
 <widget class="QWidget" name="SettingIERatio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>-40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>145</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_abajo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>85</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(194, 153, 209);</string>
   </property>
   <property name="text">
    <string>1:4.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="l_ok">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>185</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/check.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>55</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_arriba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>541</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(35, 33, 30);
     border-radius: 10px;
     color: #E3E3E3
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_ti">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>145</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_abajo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_ti">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>55</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_arriba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_ti">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>85</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(194, 153, 209);</string>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_value_ti_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>75</y>
     <width>31</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(194, 153, 209);</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>3</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 1px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>l_title</zorder>
  <zorder>l_value</zorder>
  <zorder>l_ok</zorder>
  <zorder>l_up</zorder>
  <zorder>l_down</zorder>
  <zorder>l_down_ti</zorder>
  <zorder>l_up_ti</zorder>
  <zorder>l_value_ti</zorder>
  <zorder>l_value_ti_2</zorder>
  <zorder>label_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelLongPressAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabellongpressanimated.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
